### DISCUSSION TITLE
GIMP Window resizing

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#610] [Q&A]
EoS v8.0.

I've got both Pantheon Tweaks and GIMP installed via flatpak.

The minimize button is inop. I'm not finding this in any other app.

Any thoughts?

**Comment by ryonakano** [comment]
Could you elaborate on what is your issue and your expectation? And if possible, could you attach some screenshots to show your issue?

Did you mean you added the minimize button to window controls using Pantheon Tweaks, and it doesn't appear in GIMP, while it appear in other apps?

**Comment by doobes** [comment]
LOL,

Never mind.  I toggle single wide mode to multi window mode a couple of times  and now it works.
